### DISCUSSION TITLE
登录时提示Network Error怎么办？网络明明是正常的

### DISCUSSION THREAD

**Discussion by Gan37** [ZuodaoTech/everyone-can-use-english#902] [Q&A] [answered]

**Comment by haikung** [comment]
显示网络错误，一直登录不上，下载新版本重新安装也不行

**Comment by an-lee** [comment, marked as answer]
参考这个： [总是报网络错误怎么办？](https://1000h.org/enjoy-app/faq.html#%E6%80%BB%E6%98%AF%E6%8A%A5%E7%BD%91%E7%BB%9C%E9%94%99%E8%AF%AF%E6%80%8E%E4%B9%88%E5%8A%9E)

**Reply by Gan37** [reply]
可以正常访问https://enjoy.bot/，代理开了全局也还是提示网络错误，最后改了API才有用

**Reply by an-lee** [reply]
那应该还是没有全局成功。试试在终端 ping 一个看看啥结果

```bash
ping enjoy.bot
```
